api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/IneligibleEntitlementDecisionHandler.java

```diff
@@ -0,0 +1,105 @@
+package uk.gov.dhsc.htbhf.claimant.entitlement;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.communications.DetermineEntitlementNotificationHandler;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.message.processor.ChildDateOfBirthCalculator;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimStatus;
+import uk.gov.dhsc.htbhf.claimant.model.eligibility.EligibilityAndEntitlementDecision;
+import uk.gov.dhsc.htbhf.claimant.repository.ClaimRepository;
+import uk.gov.dhsc.htbhf.claimant.service.ClaimMessageSender;
+import uk.gov.dhsc.htbhf.claimant.service.audit.EventAuditor;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static uk.gov.dhsc.htbhf.claimant.entity.CardStatus.PENDING_CANCELLATION;
+import static uk.gov.dhsc.htbhf.claimant.reporting.ClaimAction.UPDATED_FROM_ACTIVE_TO_EXPIRED;
+import static uk.gov.dhsc.htbhf.claimant.reporting.ClaimAction.UPDATED_FROM_ACTIVE_TO_PENDING_EXPIRY;
+
+@Component
+@AllArgsConstructor
+public class IneligibleEntitlementDecisionHandler {
```

Comment:
Suggest calling this something like `EligibilityDecisionHandler`

Reply:
Renamed to EligibilityDecisionHandler and moved to new eligibility package.